libr/anal/fcn.c

```diff
@@ -1142,7 +1140,7 @@ static int fcn_recurse(RAnal *anal, RAnalFunction *fcn, ut64 addr, ut64 len, int
 				}
 			} else {
 analopfinish:
-				gotoBeach (R_ANAL_RET_END);
+				gotoBeach (0);
```

Comment:
What exactly would be the meaning of 0 here?
There are multiple cases that `goto analopfinish`, maybe only one is related to the bug?

Reply:
Can you give a try to given XX tests? some of them lgtm but didnt chcked them all yet and i think some others are wrong or just wwrongly written tests